test/Function/rlexe.xml

```diff
@@ -361,6 +361,13 @@
   </test>
   <test>
     <default>
+      <files>StackArgsWithFormals.js</files>
+      <baseline>StackArgsWithFormals.baseline</baseline>
```

Comment:
Done.

---
In reply to: [67785847](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67785847) [](ancestors = 67785847)